test/unit/test-shadow-embed.js

```diff
@@ -316,6 +318,59 @@ describes.sandboxed('shadow-embed', {}, () => {
     });
   });
 
+  describe('installShadowStyle', () => {
+    let shadowRoot, shadowRoot2;
+
+    beforeEach(() => {
+      shadowRoot = document.createElement('div');
+      shadowRoot2 = document.createElement('div');
+      resetShadowStyleCacheForTesting(window);
+    });
+
+    afterEach(() => {
+      resetShadowStyleCacheForTesting(window);
+    });
+
+    it('should re-use constructable stylesheet when supported', () => {
+      try {
+        new CSSStyleSheet();
+      } catch (e) {
+        // Leave. Not supported.
+        return;
```

Comment:
nit: I believe it is possible to conditionally mark a test as skipped via `this.skip()`

Reply:
But does it short-circuit the test itself?

Reply:
Tried. It works. Changed.